tests/test_colortable.py

```diff
@@ -155,3 +155,19 @@ def test_color_table_rendering(self) -> None:
         result = str(table)
 
         assert expected == result
+
+    def test_all_themes(self) -> None:
+        """Tests rendering with all available themes"""
+        table = ColorTable(
+            ("A", "B", "C", "D", "E", "F"),
+        )
+        table.title = "Theme Test"
+        table.add_row([1, 2, 3, 4, 5, 6])
+
+        for theme_name, theme in vars(Themes).items():
+            if isinstance(theme, Theme):
+                table.theme = theme
+                result = str(table)
+                print(f"\n{theme_name} Theme:")
+                print(result)
```

Comment:
Let's remove printing from the tests:
```suggestion
```

Reply:
Ooops. Yep I'll do that today!

Reply:
Thanks!

Tip: you can also click the "Commit suggestion" button here in the GitHub UI.

Reply:
I clicked the button so I can include it in the next release. Thanks again!